dbfit-java/build.gradle

```diff
@@ -83,7 +83,75 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                delimiterType = 'normal'
+                delimiter = ';'
+                setUpTestScriptPath = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/"
+                setUpTestDbScript = setUpTestScriptPath + "create-databases-${project.name}.sql"
+                tearDownTestDbScript = setUpTestScriptPath + "drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFiles = []
+
+    task connectionFileProperties {
+        doLast {
+            copyConnectionProperties '.'
+            ext {
+                userName = ''
+                passWord = ''
+                service = ''
+                database = ''
+            }
+            if (connectionPropertiesFiles.size() > 0) {
+                def props = new Properties()
+                file(connectionPropertiesFiles[0]).withInputStream { props.load(it) }
+                ext {
+                    userName = props.username
+                    passWord = props.password
+                    service = props.service
+                    database = props.database
```

Comment:
We have such assignments (of multiple properties) in several places. But I'm not sure is it possible and how to extract it to a single place and still have the properties specific for the task.